universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -22,9 +23,11 @@
 public class ApplicantData {
   private static final String EMPTY_APPLICANT_DATA_JSON = "{ \"applicant\": {}, \"metadata\": {} }";
   private static final Locale DEFAULT_LOCALE = Locale.US;
+  private static final Joiner LIST_JOINER = Joiner.on('`');
```

Comment:
```suggestion
  private static final String LIST_DELIMITER = "`";
  private static final Joiner LIST_JOINER = Joiner.on(LIST_DELIMITER);
```

The `LIST_JOINER` and `LIST_SPLITTER` need the same token, so that should be super clear.

Reply:
Good call! Added